src/library.js

```diff
@@ -2700,6 +2713,14 @@ LibraryManager.library = {
   emscripten_longjmp: function(env, value) {
     _longjmp(env, value);
   },
+#if WASM_BACKEND == 1
+  // We need this wrapper from llvm wasm backend, but this is generated only
+  // in asm.js compilation, so.
```

Comment:
I discussed with @dschuff about this and we think it might be better to do convert all calls to emscripten_longjmp__wrapper to emscripten_longjmp and delete this hack here. Actually it was really confusing because the wasm's usage of emscripten_longjmp__wrapper and that of asm.js were different and I named it the same.